### PR TITLE
workaround for semicolon and safari 9's parseerror

### PR DESCRIPTION
**Pull request by oroce**
This intended to be a fix for #164, rather a messy workaround until this gets fixed.

What we do is tranforming this:
```
lesshat-selector {
    -lh-property: 0;
}
@-webkit-keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
@-moz-keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
@-o-keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
@keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
```

into this:
```
lesshat-selector {
    -lh-property: 0;
}
@-webkit-keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
@-moz-keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
@-o-keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
@keyframes fadein {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
[not-existing] {
    zoom: 1;
}
```

It is need since `;}` gets appended to the value which is returned by the `keyframe` function, so we are adding a (probably) nonmatching selector and the least harmful css property. :((((

### CODE REVIEW

File: mixins/keyframes/keyframes.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ var keyframes = function keyframes(value) {
 
   }
 
-  return value;
+  return value + '}\n[not-existing] {\n  zoom: 1';
```
Comment (oroce) [inline mixins/keyframes/keyframes.js]: actually we could add `foo: bar` as well.

**Comment by YouriT** [comment]
Great @oroce works fine for me :)

**Comment by listepo** [comment]
+1

**Comment by oroce** [comment]
in case someone wants to use this `less-plugin-npm-import` just install `npm install purposeindustries/lesshat#fix-keyframe-safari9`

**Comment by rahul-sekhar** [comment]
+1

**Comment by petrbrzek** [comment]
@oroce Thanks for the PR.

**Comment by yoannmoinet** [comment]
Hi @petrbrzek!
Has this been merged/built yet?
I can't found out how to get this fix from your repo.

**Comment by mboudreau** [comment]
@yoannmoinet there hasn't been a release of lesshat in 2 years...

**Comment by petrbrzek** [comment]
@yoannmoinet Sorry guys, wrong button.

**Comment by mcfedr** [comment]
what are the chances of 3.0.3?

**Comment by mboudreau** [comment]
High enough.  I think I may have time tonight to work on it.

**Comment by mboudreau** [comment]
@mcfedr version 3.0.3 released, go ahead and update your npm dependencies and see how it goes.  

Cheers.

**Comment by mboudreau** [comment]
@mcfedr Disregard 3.0.3, the build didn't package properly.  I was trying out a better way to deploy, which I think is fixed now.  Because of the major risk involved with deploying something that hasn't been done in so long, I've updated the build number to 4.0.X.  I finally got 4.0.1 working and out, however, it doesn't have the 'build' folder anymore; instead I'm just package the built less files only.

Please try it out and let me know how it goes. Cheers.

**Comment by mcfedr** [comment]
Looking good! Thanks

**Pull request merged**